src/pyramid/view.py

```diff
@@ -201,6 +201,16 @@ def my_view(context, request):
         ``view_config`` will work ONLY on module top level members
         because of the limitation of ``venusian.Scanner.scan``.
 
+        Also it will ONLY work on views present as plaintext python
+        files (*.py), ignoring all non-*.py ones which also includes
+        *.pyc and *.pyo (python precompiled byte code / optimized) files.
```

Comment:
With the change to move files into `__pycache__` and both Venusian and Pyramid going Python 3.x only, this is something we can solve in venusian. There is no reason for venusian to no longer attempt to import .pyc/.pyo files. This was mainly done to protect against the case in Python 2 where a user removed a .py file, but accidentally left a .pyc file behind.

Reply:
I'm not clear what you are requesting here for docs. Do you want to include mention of `__pycache__`? Remove .pyc/.pyo mention before it is solved in Venusian? Something else?

Reply:
I'm not requesting anything, I was merely adding commentary to help future people that stumble upon this, and leaving a note to myself to fix this in venusian.